src/modules/Dashboard/features/Participants/Participants.types.ts

```diff
@@ -3,13 +3,16 @@ import { MenuActionProps } from 'shared/components';
 import { ParticipantTag } from 'shared/consts';
 import { Encryption } from 'shared/utils';
 
+import { Invitation } from '../Applet/AddUser/AddUser.types';
+
 export type ParticipantActionProps = {
   respondentId: string | null;
   respondentOrSubjectId: string;
   email: string | null;
   secretId: string | null;
   nickname?: string | null;
   tag?: ParticipantTag | null;
+  invitation: Invitation | null;
```

Comment:
I contemplated doing this (I also added several other properties), but marking them as optional would be incorrect based on the data being returned from the BE. So I decided to go this route, and just set the properties to values that would allow the other tests to pass, since they weren't previously using them anyway